manifests/daemonset.yaml

```diff
@@ -50,9 +50,12 @@ spec:
               cpu: 10m
               memory: 64Mi
           volumeMounts:
-            - mountPath: /sys/kernel/mm/ksm
-              name: ksm
```

Comment:
Yeah KSM uses it, but it's OK, because while I dropped `/sys/kernel/mm/ksm`, I added `/sys/kernel/mm` in its place, which means node-manager can see both the `ksm` and `hugepages` subdirectories.

Reply:
Got it! Thanks for the explanation!